products/ha_memgraph/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Change Log
+
+## v0.15.0
+
```

Comment:
Do we have have to add something from memgraph v0.15.0 changes here?

Reply:
Done.